ui/components/component-library/text-field-search/text-field-search.stories.js

```diff
@@ -167,8 +164,8 @@ export default {
     },
   },
   args: {
-    showClearButton: true,
     placeholder: 'Search',
+    value: '',
```

Comment:
Need a default value for controlled inputs

## Before

<img width="1437" alt="Screenshot 2022-11-23 at 4 27 05 PM" src="https://user-images.githubusercontent.com/8112138/203672539-087d3963-59b9-412a-bb65-7a5d29525c1b.png">


## After

<img width="1440" alt="Screenshot 2022-11-23 at 4 27 46 PM" src="https://user-images.githubusercontent.com/8112138/203672547-f6c1a9d1-ad50-4a87-80c5-f107e4c5d3fd.png">